loadtest.py

```diff
@@ -1,32 +1,61 @@
 import os
 
-from molotov import setup, scenario
+from fxa.__main__ import DEFAULT_CLIENT_ID
+from fxa.core import Client
+from fxa.tests.utils import TestEmailAccount
+from fxa.tools.bearer import get_bearer_token
+from molotov import global_setup, global_teardown, setup, scenario
 
 
 # Read configuration from env
 SERVER_URL = os.getenv(
-    'URL_KINTO_SERVER',
+    'KINTO_WE_SERVER',
     "https://webextensions-settings.stage.mozaws.net").rstrip('/')
 
-#FXA_BEARER_TOKEN = os.getenv("FXA_BEARER_TOKEN")
-FXA_BEARER_TOKEN = "b188fdd9f7a3266fbbe6e28754f7f1fe7deb3b0ca1b61fae7bfc7c55c2e279cd"
-if not FXA_BEARER_TOKEN:
-    raise ValueError("Please define FXA_BEARER_TOKEN env variable.")
-
-CONNECTIONS = {}
-
 COLLECTIONS = "/v1/buckets/default/collections"
 
 STATUS_URL = "/v1/"
 VERSION_URL = "/v1/__version__"
 
-'''
-http -v GET "${SERVER_URL}/buckets/default/collections" Authorization:"Bearer ${OAUTH_BEARER_TOKEN}"
-'''
+_FXA = {}
+
+@global_setup()
+def create_account_and_token(args):
+    acct = TestEmailAccount()
+    client = Client("https://api.accounts.firefox.com")
+    session = client.create_account(
+        acct.email,
+        'MySecretPassword'
```

Comment:
To make this a safe as possible, I would replace this hardcoded string with a random password. Even though this account will only last for the duration of the test.